### PR TITLE
`pr-reminder`: add hvidosil and the new prow location

### PR DESCRIPTION
**Pull request by smg247**

### CODE REVIEW

File: core-services/pr-reminder/_config.yaml [hunk 1]
```diff
@@ -12,13 +12,15 @@ teams:
...
   - lxia
+  - hvidosil
   teamNames:
   - test-platform
   repos:
   - openshift/ci-tools
   - openshift/ci-docs
   - openshift/release
   - kubernetes/test-infra
+  - kubernetes-sigs/prow
```
Comment (Prucek) [inline core-services/pr-reminder/_config.yaml:23]: what does this? how is this useful?

**Review by Prucek — APPROVED** [review]

File: core-services/pr-reminder/_config.yaml (hunk 1 above)
Comment (smg247) [reply core-services/pr-reminder/_config.yaml:23]: it defines the repos that our team receives pr-reminder digests for. Prow moved awhile back, and this wasn't updated.

Comment (Prucek) [reply core-services/pr-reminder/_config.yaml:23]: so if I am asked to review in prow now, will I get notified?

Comment (smg247) [reply core-services/pr-reminder/_config.yaml:23]: yes

**Pull request merged**